### PR TITLE
Build latest OSS docs automatically.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of animals._

arctic fox

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/fAZrlKcu

## Description
_The commit messages say what you did; this should explain why and how._

- Added build script to download latest versions of OSS projects, copy docs into OddLeventy, and build site.
- Updated Netlify to use new `yarn prod` cmd

## Impacted areas of website
_Provide URLs and/or a description of where to look._

- https://5e4d63085d8bc5000d788023--oddleventy.netlify.com/susy/docs/
- https://5e4d63085d8bc5000d788023--oddleventy.netlify.com/herman/docs/
- https://5e4d63085d8bc5000d788023--oddleventy.netlify.com/true/docs/
- https://5e4d63085d8bc5000d788023--oddleventy.netlify.com/accoutrement/docs/

---

## Reviewer tasks:

- [ ] Pulled branch and ran code locally (or viewed on preview deployment, if applicable)
- [ ] Reproduced the issue to review in the browser
- [ ] Reviewed code
- [ ] Updated Trello card (if applicable)

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
@wlonk Any thoughts/feedback on https://github.com/oddbird/oddleventy/pull/24/files#diff-9d10158d59cee97248164fe5ad5c1d42 ?

File: bin/update-oss-docs [hunk 1]
```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-yarn global add susy sass-true sassdoc-theme-herman accoutrement --no-lockfile --ignore-scripts --no-bin-links --global-folder temp
+exit_on_error() {
+    exit_code=$1
+    if [ $exit_code -ne 0 ]; then
+        exit $exit_code
+    fi
+}
```
Comment (wlonk) [inline bin/update-oss-docs]: You can avoid this whole function by adding ` -e` to the shebang line. That will make any command exiting with a nonzero code exit the whole script with that code.

Comment (jgerigmeyer) [reply bin/update-oss-docs]: Yeah, I had that at first, but the decided I think we only want to bail if the first command fails. But I'm not really sure, maybe I should switch back.

Comment (wlonk) [reply bin/update-oss-docs]: Not vital, and I understand that reasoning. Still 👍

**Pull request merged**